### DISCUSSION TITLE
Support for "if" option for "attribute" without a block?

### DISCUSSION THREAD

**Discussion by kapso** [okuramasafumi/alba#331] [Feedback]
I can do this using `attributes` -> `attributes :slug, if: ->(record) { record.countable? }`

...but when I do this using `attribute` -> `attribute :slug, if: ->(record) { record.countable? }`

I get the following error - `raise ArgumentError, 'No block given in attribute method' unless block`

**Comment by okuramasafumi** [comment]
That's because `attribute` MUST take a block.
https://github.com/okuramasafumi/alba#simple-serialization-with-root-key
